cs/src/core/Index/Interfaces/IFasterKV.cs

```diff
@@ -103,8 +100,37 @@ public interface IFasterKV<Key, Value, Input, Output, Context> : IDisposable
         /// <returns>Whether all pending operations have completed</returns>
         bool CompletePending(bool wait);
 
+        #endregion
+        */
+
+        #region New Session Operations
+        /// <summary>
+        /// Start new client session (not thread-specific) with FASTER.
+        /// Session starts in dormant state.
+        /// </summary>
+        /// <param name="sessionId"></param>
+        /// <param name="supportAsync"></param>
+        /// <returns></returns>
+        ClientSession<Key, Value, Input, Output, Context, Functions> NewSession(string sessionId = null, bool supportAsync = true);
 
-        /* Recovery */
+        /// <summary>
+        /// Continue client session with FASTER
```

Comment:
nit: it's probably better to clarify whether this is used exclusively for recovery from failure, or some how a session can be "suspended" as part of normal operation